wrappers/java/src/main/java/tech/pegasys/era/EthereumRegistrationAuthority.java

```diff
@@ -1,12 +1,22 @@
-package tech.pegasys.era;
+/*
+ * Copyright 2019 ConsenSys AG.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ */
+package era;
```

Comment:
Is this package change correct?

Reply:
No, the package is not correct. Code wasn't ready for code review so I have resolved this for now.